<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicFileViewer</class>
 <widget class="QDialog" name="BasicFileViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View file</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblArchiveInfo">
     <property name="text">
      <string>Archive:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblFileInfo">
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="txtContents">
     <property name="openLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtContentsPlain">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDone">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
